s4sim/hardware/config.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2020 CMB-S4 Collaboration.
+# Copyright (c)
```

Comment:
Any particular reason for trimming the copyright statement?

Reply:
just an old copy before we had those likely